syncer/syncer.go

```diff
@@ -165,6 +172,11 @@ func (s *syncer) bulkSyncWithPeer(peerID string, newBlockCallback func(*types.Bl
 	for {
 		select {
 		case block, ok := <-blockCh:
+			// safe check
+			if block.Number() == 0 {
+				continue
+			}
+
 			if !ok {
```

Comment:
if !ok needs to be before if block.Number == 0, or we can get null pointer error

Reply:
Good catch.
Fixed 67e14fffdf70d5f699d1d272d2f37906497bce5f